quixstreams/state/rocksdb/windowed/partition.py

```diff
@@ -81,19 +72,9 @@ def write(
     ):
         batch = WriteBatch(raw_mode=True)
```

Comment:
I know it was already like this, but that looks strange to ignore the batch argument and always initialize it from scratch. Could you check if everything is alright there?

Reply:
Same here https://github.com/quixio/quix-streams/pull/591#discussion_r1825610733